projects/core/components/button/button.component.ts

```diff
@@ -22,11 +22,10 @@ import {
     TuiHoveredService,
     watch,
 } from '@taiga-ui/cdk';
-import {TuiAppearance} from '@taiga-ui/core/enums';
-import {TuiSizeS, TuiSizeXL, TuiSizeXS} from '@taiga-ui/core/types';
-import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+import {TuiSizeS} from '@taiga-ui/core/types';
 import {Observable} from 'rxjs';
 import {takeUntil} from 'rxjs/operators';
+import {ButtonOptions, TUI_BUTTON_OPTIONS} from './button-options';
 
 @Component({
```

Comment:
Let's add interface implementation so the types are more apparent.

Reply:
done